python/cloudtik/core/_private/cluster/cluster_operator.py

```diff
@@ -2125,13 +2125,18 @@ def _start_proxy(config: Dict[str, Any],
 def _start_proxy_process(head_node_ip, config,
                          bind_address: str = None):
     proxy_info_file = get_proxy_info_file(config["cluster_name"])
+    cmd = "ssh -o \'StrictHostKeyChecking no\'"
 
     auth_config = config["auth"]
     ssh_proxy_command = auth_config.get("ssh_proxy_command", None)
-    ssh_private_key = auth_config.get("ssh_private_key", None)
-    ssh_user = auth_config["ssh_user"]
+    if config["provider"]["type"] == "kubernetes":
```

Comment:
To do special check according to the specific provider type should avoided. This is the principle for design of provider.